### DISCUSSION TITLE
v3.0.0 Statistics

### DISCUSSION THREAD

**Discussion by oldsparks** [artisan-roaster-scope/artisan#1656] [Q&A]
Thanks for all the work on Artisan and the current new features! I just installed and have several roasts to accomplish today. 

I'm curious if there is a way to show or hide the statistics summary on the graph? I find the summary quite useful when reviewing previous roasts, however I also often find I'd like it to go away if I'm sharing a profile as an image or recording BBP and have "show all" set. I don't know about anyone else but I know I'd be toggling it quite often. I love that I can easily toggle background profiles as well as the ET RoR curve for example and I'd like to be able to toggle the summary in the same way.

**Comment by CarefreeBuzzBuzz** [comment]
No toggle butStatisticsartisan-scope.org___________________________Michael Herbert, CarefreeBuzzBuzzOn Aug 7, 2024, at 8:54 AM, oldsparks ***@***.***> wrote:﻿
Thanks for all the work on Artisan and the current new features! I just installed and have several roasts to accomplish today.
I'm curious if there is a way to show or hide the statistics summary on the graph? I find the summary quite useful when reviewing previous roasts, however I also often find I'd like it to go away if I'm sharing a profile as an image or recording BBP and have "show all" set. I don't know about anyone else but I know I'd be toggling it quite often. I love that I can easily toggle background profiles as well as the ET RoR curve for example and I'd like to be able to toggle the summary in the same way.

**Reply by oldsparks** [reply]
Hi Michael, 

So how can I make that a formal feature request for the future?

**Comment by roasterdave** [comment]
To show or hide the Summary Statistics go to Config>> Statistics>> Stats Summary tab and tick or untick "Show Summary".  At this time there is no shortcut key.

![image](https://github.com/user-attachments/assets/1ed83fa6-92a0-48f1-99ca-87f57bedc42d)
